SwiperArrows.js

```diff
@@ -35,18 +35,36 @@ class SwiperArrows {
                 e.preventDefault();
 
                 if (this._clickSpacePrevious.classList.contains("active")) {
-                    this.swiper.moveRight(true);
+                    this.swiper.moveLeft(true);
                 }
 
                 this._runEventListeners('clickSpacePreviousClicked');
             };
 
             this._clickSpacePrevious.addEventListener('click', this._clickSpacePreviousOnClickListener);
         }
+
+        this.swiper.addEventListener('move', () => {
```

Comment:
I don't think there's a need to even subscribe "move" event for infinite sliders. Less code and better performance.